### PR TITLE
added style to cv

### PR DESCRIPTION
**Pull request by Javier1177**

### CODE REVIEW

File: staff/javier-lopez/cv/styles.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+figure img{
+    border-radius: 100px;
+}
+
+body{
```
Comment (mikelpmc) [inline staff/javier-lopez/cv/styles.css]: Following the hierarchy this should be on top of the css file

File: staff/javier-lopez/cv/styles.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+.content{
+    border: 2px solid black;
+}
+main{
+    width: 70%;
+    float: left;
+}
+
+#name{
```
Comment (mikelpmc) [inline staff/javier-lopez/cv/styles.css:25]: When possible better use classes over ids

File: staff/javier-lopez/cv/styles.css (hunk 1 above)
Comment (Javier1177) [reply staff/javier-lopez/cv/styles.css]: Updated!

File: staff/javier-lopez/arrays/main.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+//Add all elements
+
+function addAllElements(array){
+    res = 0;
```
Comment (alexgbor) [inline staff/javier-lopez/arrays/main.js]: var <--- missing

Comment (Javier1177) [reply staff/javier-lopez/arrays/main.js]: Fixed!

File: staff/javier-lopez/cv/styles.css (hunk 2 above)
Comment (Javier1177) [reply staff/javier-lopez/cv/styles.css:25]: Okay!

File: staff/javier-lopez/arrays/scripts/sort.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+function sort(array, reverse) {
+    // TODO return array sorted alphabetically / numerically (use of Array.prototype.sort is forbidden)
+    var ordenedArray = [];
+    for(var i = 0; i<array.length;i++){
```
Comment (carlotavicens) [inline staff/javier-lopez/arrays/scripts/sort.js:4]: falta omplir el for

File: staff/javier-lopez/beers-app/index.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (maiderhernandorena) [inline staff/javier-lopez/beers-app/index.html:7]: You should put title to your files : D
```
<head>
<title>Beers App</title>
</head>
```

File: staff/javier-lopez/beers-app/scripts/main.js [hunk 6]
```diff
@@ -67,23 +63,15 @@ ResultsList.prototype.onItemClick = function (callback) {
  * 
  * @param {string} title The item title
  * @param {string} info The information about an item
- * @param {[number]} coords The geodesic coordinates for google maps
+ * @param {[number]} abv The graduation of the beer
```
Comment (maiderhernandorena) [inline staff/javier-lopez/beers-app/scripts/main.js]: If you put as parameters (title, abv, description), try to put in the comments in the same order. Then, the person who reads your code could understand better what you are using below. : D

File: staff/javier-lopez/arrays-native/scripts/copyWithin.test.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+describe('Array.prototype.copyWithin', function () {
+    var firstArray;
```
Comment (maiderhernandorena) [inline staff/javier-lopez/arrays-native/scripts/copyWithin.test.js:4]: i understand you have copy and reuse other files code, but you may change var names to be more semantic. 
`var firstArray;  => var array;  | var nums;`
explanation: there is no secondArray to need a firstArray.

**Pull request merged**